packages/server/graphql/private/mutations/setIsFreeMeetingTemplate.ts

```diff
@@ -0,0 +1,43 @@
+import getRethink from '../../../database/rethinkDriver'
+import {MutationResolvers} from '../resolverTypes'
+
+const setIsFreeMeetingTemplate: MutationResolvers['setIsFreeMeetingTemplate'] = async (
+  _source,
+  {isFree, templateIds, templateNames}
+) => {
+  const r = await getRethink()
+  // VALIDATION
+  if (!templateIds?.length && !templateNames?.length) {
+    return {error: {message: 'Must provide template ids or names'}}
+  }
+  if (templateIds?.length && templateNames?.length) {
+    return {error: {message: 'Please provide template ids or names, not both'}}
+  }
+
+  // RESOLUTION
+  if (templateIds?.length) {
+    const updatedTemplateIds = await r
+      .table('MeetingTemplate')
+      .getAll(r.args(templateIds))
+      .update(
+        {isFree},
+        {returnChanges: true}
+      )('changes')('new_val')('id')
+      .default(null)
+      .run()
+    return {templateIds: updatedTemplateIds}
+  } else {
+    const updatedTemplateIds = await r
+      .table('MeetingTemplate')
+      .filter((row) => r.expr(templateNames).contains(row('name')))
+      .update(
+        {isFree},
+        {returnChanges: true}
+      )('changes')('new_val')('id')
+      .default(null)
+      .run()
+    return {templateIds: updatedTemplateIds}
```

Comment:
If no rows updated, for example they are already `isFree: true`, no ids returned and no result returned in mutation even if templates exists:

<img width="1294" alt="image" src="https://user-images.githubusercontent.com/466991/198256955-722972f8-1745-4e08-9563-d07d0c184b10.png">

Is it expected behaviour?

Reply:
Yep! I wanted to let the user know if a template has been updated, so I only wanted to return the updated templates